modules/tinymce/src/plugins/table/test/ts/browser/InlineEditorInsideTableTest.ts

```diff
@@ -35,13 +39,97 @@ describe('browser.tinymce.plugins.table.InlineEditorInsideTableTest', () => {
     inline: true,
     base_url: '/project/tinymce/js/tinymce',
     plugins: 'table',
-    menubar: false,
+    menubar: 'table',
     statusbar: false
   }, setupElement, [ Plugin, Theme ]);
 
+  const focusEditor = (editor: Editor) => {
```

Comment:
Why is this needed? Can we just use `editor.focus()` instead, as we generally don't like to use internal core modules like this?

Additionally, this seems to be called in every test, so can we move this to a `beforeEach` instead please?

- [ ] Use `editor.focus()` and consider a `beforeEach` hook instead please.